### PR TITLE
Create definitions for gulp-jasmine@2.4 

### PR DESCRIPTION
**Pull request by andypyrope**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/gulp-jasmine/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// Type definitions for gulp-jasmine 2.4
+// Project: https://github.com/sindresorhus/gulp-jasmine#readme
+// Definitions by: Andrey Lalev <https://github.com/andypyrope>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="node" />
+/// <reference types="jasmine" />
+
+declare namespace gulpJasmine { }
```
Comment (plantain-00) [inline types/gulp-jasmine/index.d.ts]: If I'm right, the namespace can be removed.

**Review by plantain-00 — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
@andy-ms @RyanCavanaugh any idea why Travis isn't getting run here?

File: types/gulp-jasmine/index.d.ts (hunk 1 above)
Comment (andypyrope) [reply types/gulp-jasmine/index.d.ts]: Thanks for mentioning this. Honestly, I thought it was a necessary workaround for creating a definition for a module-function but fortunately I was wrong.

**Comment by andypyrope** [comment]
No wonder this hasn't been approved yet. I have no idea why Travis forgot to do that, maybe it was under too much stress due to many pull requests, but it seems to have been run now.

@DanielRosenwasser thanks for addressing the issue found by @plantain-00 . I was going to push a fix and then I noticed you had already done that.

**Pull request merged**